02-electric/electric-section-one/electric-section-one.css

```diff
@@ -1,14 +1,28 @@
 .electric-section-one-container {
   padding: var(--container-padding-mobile);
+  display: grid;
+}
+
+@media (min-width:500px) {
+  .electric-section-one-container {
+    padding: var(--container-padding-desktop);
+  }
 }
 
 .electric-section-one-subcontainer {
   display: grid;
   grid-template-columns: 1fr;
   justify-content: center;
-  row-gap: var(--items-gap-mobile);
+  row-gap: 30px;
   padding: var(--subcontainer-padding-mobile);
   border: solid 1px black;
+  background-color: var(--level1-background-color);
+}
+
+@media (min-width:500px){
+  .electric-section-one-subcontainer{
+    padding: var(--subcontainer-padding-desktop);
+  }
 }
 
 .electric-section-one-subcontainer > * {
```

Comment:
remove this completely.
add to the top a list of all the elements you want to have the temporary border

Reply:
done